bin/regenerate-testdata.sh

```diff
@@ -6,7 +6,7 @@ SALESFORCE=./test_data/salesforce.csv
 SALESFORCE_SORTED=./test_data/salesforce_sorted.csv
 EXPECTED_COUNTS=./test_data/expected.json
 
-COUNT=4000
+COUNT=5000000
```

Comment:
So, the travis test times out with 5M records. Maybe 50K?

Reply:
Shit, I didn't mean 5M. I meant to revert this.